### PR TITLE
Added test for API Provider with conditional flow

### PR DESCRIPTION
**Pull request by mmuzikar**
//test: `@api-provider-conditional-flow or @conditional-flow`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mmuzikar** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/api-provider.feature [hunk 1]
```diff
@@ -1151,3 +1151,73 @@ Feature: API Provider Integration
     Then wait until integration "TODO Integration base path" gets into "Running" state
     When select the "TODO Integration base path" integration
     Then verify the displayed API Provider URL matches regex ^https://i-todo-integration-base-path-syndesis.*/api$
+
+  @api-provider-conditional-flow
+  Scenario: Base path in API Provider URL
```
Comment (asmigala) [inline ui-tests/src/test/resources/features/integrations/api-provider.feature]: Change copy-pasta name :D

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Conditional flows - content base routing | Conditional flows - delete step 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Conditional flows - content base routing | Conditional flows - functional test 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/java/io/syndesis/qe/steps/other/ConditionalFlowSteps.java [hunk 2]
```diff
@@ -112,7 +113,8 @@ public void configureConditionOnPosition(int index) {
     @When("^return to primary flow from integration flow$")
     public void goBackToPrimaryFlow() {
         openDropdownWithConditions();
-        new CommonSteps().clickOnLink("Back to Primary Flow");
+        $(EditFlowStepElements.FLOW_BACK_BUTTON).shouldBe(visible).click();
+        //new CommonSteps().clickOnLink("Back to Primary Flow");
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/steps/other/ConditionalFlowSteps.java]: why is this commented? Can we remove it?

Comment (mmuzikar) [reply ui-tests/src/test/java/io/syndesis/qe/steps/other/ConditionalFlowSteps.java]: Yep forgot to delete it after I confirmed the new way worked, thx for noticing

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**